core/src/util/usLDAPExpr_p.h

```diff
@@ -0,0 +1,180 @@
+/*=============================================================================
```

Comment:
I searched online, and found that it seems not possible with github UI. 
Summary of changes:
- use usProperties class instead of usServicePropertiesImpl class.